[2025-01-06T23:34:24.141+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_insert_log.sql_md_account_d manual__2025-01-06T23:28:13.745960+00:00 [queued]>
[2025-01-06T23:34:24.168+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_insert_log.sql_md_account_d manual__2025-01-06T23:28:13.745960+00:00 [queued]>
[2025-01-06T23:34:24.169+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 2
[2025-01-06T23:34:24.208+0000] {taskinstance.py:2191} INFO - Executing <Task(SQLExecuteQueryOperator): sql_md_account_d> on 2025-01-06 23:28:13.745960+00:00
[2025-01-06T23:34:24.227+0000] {standard_task_runner.py:60} INFO - Started process 529 to run task
[2025-01-06T23:34:24.248+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_insert_log', 'sql_md_account_d', 'manual__2025-01-06T23:28:13.745960+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/dag_insert_log.py', '--cfg-path', '/tmp/tmpwmle42wm']
[2025-01-06T23:34:24.250+0000] {standard_task_runner.py:88} INFO - Job 30: Subtask sql_md_account_d
[2025-01-06T23:34:24.432+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_insert_log.sql_md_account_d manual__2025-01-06T23:28:13.745960+00:00 [running]> on host 22295d32c47d
[2025-01-06T23:34:24.636+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='saida' AIRFLOW_CTX_DAG_ID='dag_insert_log' AIRFLOW_CTX_TASK_ID='sql_md_account_d' AIRFLOW_CTX_EXECUTION_DATE='2025-01-06T23:28:13.745960+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-06T23:28:13.745960+00:00'
[2025-01-06T23:34:24.650+0000] {sql.py:276} INFO - Executing: CALL ds.stage_to_md_account_d();
[2025-01-06T23:34:24.692+0000] {base.py:83} INFO - Using connection ID '***-db' for task execution.
[2025-01-06T23:34:24.736+0000] {base.py:83} INFO - Using connection ID '***-db' for task execution.
[2025-01-06T23:34:25.062+0000] {sql.py:450} INFO - Running statement: CALL ds.stage_to_md_account_d();, parameters: None
[2025-01-06T23:34:25.147+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_insert_log, task_id=sql_md_account_d, execution_date=20250106T232813, start_date=20250106T233424, end_date=20250106T233425
[2025-01-06T23:34:25.208+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-01-06T23:34:25.262+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
